frontend/src/metabase/query_builder/components/DataSelector/DataSelectorDataBucketPicker/DataSelectorDataBucketPicker.module.css

```diff
@@ -1,6 +1,5 @@
 .DataBucketList {
   width: 300px;
-  padding: 4px 8px 12px;
```

Comment:
This padding is moved to the parent container, as this is overridden by this line as non-Mantine SelectList uses `ul` underneath, cc @oisincoveney 

https://github.com/metabase/metabase/blob/c7b50a14267e2fa065b759453e9c9998e3fa5833/enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx#L68